### ISSUE TITLE
CommonJS 'dist/findalab.js' and 'src/js/findalab.js' don't behave same

### ISSUE THREAD

**Issue by rashidsaf** [Medology/findalab#179]
## Describe the bug
 Using Findalab in any other project doesn't work with CommonJS version 3.0.5. The reason is `dist` and `src` versions are either not the same or don't behave the same. If you change the `import FindALab from "./dist/findalab";` to `import FindALab from "./src/js/findalab";` everything loads and works

**Comment by rashidsaf** [comment]
* Find the problem and fixed the issue
* Tested and created a PR

**Comment by rashidsaf** [comment]
- Scored with @manuerumx and @izazueta 
- Moved to Prep QA

**Comment by rashidsaf** [comment]
Waiting for **QA**

**Comment by juliohl** [comment]
### QA - Local #180 
I could run a sanity test for "Find a lab" feature, however, for this PR there is no local UI that can be tested at least from that perspective. @Chekote what do you think can be done in here to test? I'm thinking the only way to test this is in prod; unless the whole feature can be enabled in local so I can do a Sanity here. Please let me know so we can move this one.

**Comment by rashidsaf** [comment]
Waiting for QA

**Comment by juliohl** [comment]
### QA - Local (#180)
It is working as expected, tested by:
- Searching a non-numeric string.
- Searching without any value in the zip code field
- Searching a valid code
- Searching a numeric non-valid zip code
- This was done in the latest Chr, FF, IE

Moving to Deploy& Test

**Comment by rashidsaf** [comment]
- Merged and tested
- Created new release on Github **3.0.6**

**Comment by juliohl** [comment]
### QA Prod
Verified in prod, all is working as expected, moving to **Approval**.

**Comment by Chekote** [comment]
I'm not confident enough to PO Approve this. @BKorenek would you mind taking a look at the PR and let me know if the problem described here should be fixed by the changes?

**Comment by BKorenek** [comment]
@rashidsaf can you do a PR to overwrite your recent implementation on STDChecks end, to verify the functionality.

**Comment by rashidsaf** [comment]
- Created a PR to fixate the jQUery version on findalab
- Going to create another PR to update the findalab version on STCheck and change the import path to `dist`

**Comment by rashidsaf** [comment]
- Moved to Pre Production QA,

**Comment by juliohl** [comment]
### QA Local
Tested on http://findalab.local/user-location.php, seem it is behaving as expected. Moving to **D&T**

**Comment by juliohl** [comment]
### QA Local (#184)
Tested on http://findalab.local/user-location.php, seems it is behaving as expected. Moving to **D&T**

**Comment by jose-ochoa** [comment]
Wil test today

**Comment by rashidsaf** [comment]
- Tested and everything works fine on FIndalab
- Will try to bump the version on STDCheck and create a PR

**Comment by rashidsaf** [comment]
- Worked on this yesterday, still having some problems caused by legacy code and new commonJs having conflicts over minifed dist files
- Will try to make it work today if not, I will keep the old implementation

**Comment by rashidsaf** [comment]
[Standup]
- Created a new followup PR for findalab

**Comment by rashidsaf** [comment]
### Tuesday Standup

Waiting for QA

**Comment by juliohl** [comment]
## QA Local
Done with a Sanity on the component, it is working as expected, moving to **D&T**.

**Comment by fiyyazp** [comment]
will merge today

**Comment by rashidsaf** [comment]
### Wednesday
Blocked by Johnnie's PR https://github.com/Medology/stdcheck.com/pull/10175

**Comment by rashidsaf** [comment]
### Friday
This issue unblocked and I put my PR under review. Waiting for code review

**Comment by rashidsaf** [comment]
### Monday
The issue is blocked again with the issue, seems like there were some problems, waiting to be unblocked

**Comment by rashidsaf** [comment]
Yesterday I got unblocked so I rebased my PR and got some failing tests. Will try to fix them and retry

**Comment by rashidsaf** [comment]
Created a PR for findalab repo to sync the dist files. After get one gets merged, will update my main PR

**Comment by kabbatehl** [comment]
### QA Local #207 
Nothing to test.  Moving to deploy and test.

**Comment by rashidsaf** [comment]
Merged the PR for findalab and waiting for code review for STDCheck version update

**Comment by rashidsaf** [comment]
Waiting for CR

**Comment by kabbatehl** [comment]
### QA Local #10222
Tried testing this but got an error at http://www.stdcheck.local/order.
![screenshot-nimbus-capture-2020 07 13-17_19_06](https://user-images.githubusercontent.com/62027147/87360115-05d1ed00-c52f-11ea-8500-773e9ede204e.png)

**Comment by kabbatehl** [comment]
### QA - Local (#10222)
It is working as expected, tested by:

Searching a non-numeric string.
Searching without any value in the zip code field
Searching a valid code
Searching a numeric non-valid zip code
This was done in the latest Chr, FF, IE
Moving to Deploy& Test

**Comment by rashidsaf** [comment]
Passed QA, so will deploy today

**Comment by rashidsaf** [comment]
There are some updates and Laravel 5.7 upgrade that may affect the findalab, so I will merge master and get this quick QA tested today

**Comment by rashidsaf** [comment]
Rebased and deployed. Tested findalab pages and moved to the Production QA

**Comment by kabbatehl** [comment]
QA - Prod (#10222)
It is working as expected, tested by:

Searching a non-numeric string.
Searching without any value in the zip code field
Searching a valid code
Searching a numeric non-valid zip code
This was done in the latest Chr, IE, ios safari

Tested on all pages that use find a lab. Order page, Stdcheck test center, better lab, and consultation request.
Moving to

**Comment by Chekote** [comment]
@BKorenek would you mind taking another look at this and let me know if the problem described here is resolved?

**Comment by BKorenek** [comment]
@rashidsaf I feel like the solution to this could have been a little more in line with the norm. Since it's a JS package you should be able to declare the root script to import from. E.g. dist/index.js or something like that. This would allow us to just import the findalab directly. `import Findalab from 'findalab'`. We should also strip out the compilation entirely. minification and traspilation should be done on the project that is importing the script, that way if we have a specific support that is necessary on a specific project, we could then insure we have the proper polyfills on that project.

**Comment by rashidsaf** [comment]
@BKorenek that is the current implementation, the minified file not used in Stdcheck and the syntax is like `import Findalab from 'findalab'`

**Comment by Chekote** [comment]
@BKorenek can I approve this? Or is there still something that needs to be resolved?

**Comment by BKorenek** [comment]
@Chekote This is good to go.
